terraform/main.tf

```diff
@@ -0,0 +1,161 @@
+terraform {
+  backend "s3" {
+    bucket = "serokell-gemini-tfstate"
+    dynamodb_table = "serokell-gemini-tfstate-lock"
+    encrypt = true
+    key    = "gemini/terraform.tfstate"
+    region = "eu-west-2"
+  }
+  ## Prevent unwanted updates
+  required_version = "~> 0.12.29" # Use nix-shell or nix develop
```

Comment:
`=` instead of `~>` should be better here, so we don't accidentally update the state to 0.12.30 for example